packages/neutron-sdk/src/interchain_queries/types.rs

```diff
@@ -2,15 +2,23 @@ use crate::bindings::types::StorageValue;
 use crate::errors::error::NeutronResult;
 use crate::NeutronError;
 use cosmos_sdk_proto::cosmos::base::v1beta1::Coin as CosmosCoin;
-use cosmos_sdk_proto::cosmos::staking::v1beta1::{Delegation, Validator};
+use cosmos_sdk_proto::cosmos::distribution::v1beta1::FeePool as CosmosFeePool;
+use cosmos_sdk_proto::cosmos::gov::v1beta1::Proposal as CosmosProposal;
+use cosmos_sdk_proto::cosmos::staking::v1beta1::{Delegation, Validator as CosmosValidator};
 use cosmwasm_std::{from_binary, Addr, Coin, Decimal, Uint128};
 use prost::Message as ProstMessage;
 use schemars::JsonSchema;
 use schemars::_serde_json::Value;
 use serde::{Deserialize, Deserializer, Serialize, Serializer};
 use std::ops::Div;
+use std::str;
```

Comment:
This import is unused. In fact, I don't like importing `std::str` since it collides with built-in type `str`.